docs/graphcast/radios/poi-radio.md

```diff
@@ -12,37 +12,93 @@ The key requirement for an Indexer to earn indexing rewards is to submit a valid
-
-#### Using pre-built image
-
-This is the option you should go for if you're just looking to run the POI Radio as part of your Indexing stack.
+### Quickstart
 
 ##### Steps
 
-1. Pull the image
+1. Pull the POI Radio image
 
 ```bash
 docker pull ghcr.io/graphops/poi-radio:latest
 ```
 
-2. Run the image, providing the required environment variables
+2. Run the image, providing the required environment variables. Here's a sample testnet configuration:
 
 ```bash
-docker run -e REGISTRY_SUBGRAPH="https://api.thegraph.com/subgraphs/name/hopeyen/gossip-registry-test" -e NETWORK_SUBGRAPH="https://gateway.testnet.thegraph.com/network" -e GRAPH_NODE_STATUS_ENDPOINT="http://host.docker.internal:8030/graphql" -e PRIVATE_KEY="your operator address private key" -e ETH_NODE="an ethereum rpc node url" ghcr.io/graphops/poi-radio
+docker run \
+-e ETH_NODE="ETH_NODE" \
+-e PRIVATE_KEY="GRAPHCAST_ID_PRIVATE_KEY" \
+-e GRAPH_NODE_STATUS_ENDPOINT="http://host.docker.internal:8030/graphql" \
+-e REGISTRY_SUBGRAPH="https://api.thegraph.com/subgraphs/name/hopeyen/graphcast-registry-goerli" \
+-e NETWORK_SUBGRAPH="https://gateway.testnet.thegraph.com/network" \
+-e GRAPHCAST_NETWORK="testnet" \
+-e RUST_LOG="off,hyper=off,graphcast_sdk=debug,poi_radio=debug" \
+ghcr.io/graphops/poi-radio
 ```
 
 :::tip
-The `PRIVATE_KEY` here is for your Graphcast Operator address, which is entirely separate from your Indexer Operator address. The Graphcast Operator address is authorised using our [Registry contract](https://goerli.etherscan.io/address/0xBFdA8191D1ec09bB8ADc138DAbf413d00DAfb6c8). You need to use your Indexer wallet to call the `setGossipOperator` function, providing the operator address.
+The `PRIVATE_KEY` here is for your Graphcast ID. The Graphcast ID address is authorised using our **Registry contract**, we have [one for mainnet](https://etherscan.io/address/0x89f97698d6006f25570cd2e31737d3d22aedcbcf) and [one for testnet](https://goerli.etherscan.io/address/0xe88461436D30CAc2d69b70871EC2cdDC32083aed) (on Goerli). You need to use your Indexer or Operator wallet to first call the `setStaking` function, providing the address of the staking contract, and then call the `setGraphcastIDFor` function, providing the Indexer address, as well as the address you wish to use as a Graphcast ID.
 :::
 
-To see the full list of environment variables you can provide, check out the [Configuration](#configuration) section.
+In the configuration table below is the full list of environment variables you can use, along with example values.
+
+## Configuration
+
+### Mainnet
+
+| Name                                  | Example                                                                                  |
+| ------------------------------------- | ---------------------------------------------------------------------------------------- |
+| `ETH_NODE`                            | `https://mainnet.infura.io/v3/<PROJECT_ID>`                                              |
```

Comment:
I know this is an example but I'm a bit confused by <PROJECT_ID>... could be switched to `API_KEY`?